From 3d33a73b58cabc9ed784b450409f07f3fd755844 Mon Sep 17 00:00:00 2001
From: Ian Denhardt <ian@zenhack.net>
Date: Tue, 6 Sep 2022 11:57:19 -0400
Subject: [PATCH 28/28] Increase max socket.io buffer size.

This is a workaround for:

    https://github.com/ether/etherpad-lite/issues/4951;

...10K is not a lot of text. 250K should trigger a lot
less.
---
 settings.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/settings.json b/settings.json
index 135222e1..ef46049e 100644
--- a/settings.json
+++ b/settings.json
@@ -502,7 +502,7 @@
      * value to work properly, but increasing the value increases susceptibility
      * to denial of service attacks (malicious clients can exhaust memory).
      */
-    "maxHttpBufferSize": 10000
+    "maxHttpBufferSize": 250000
   },
 
   /*
-- 
2.37.3

